CLAUDE.md

```
@@ -0,0 +1,159 @@
+melos run analyze        # dart analyze --fatal-infos, every package
+melos run format
+melos run generate:flutter   # build_runner, for *.g.theme.dart and friends
+melos run test:all
+```
+
+## The design system lives in another repository
+
+`stream_core_flutter` is the design system and is **not** in this repo. It is a
+git dependency pinned in the root `pubspec.yaml`, so its source is at:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)